src/Sarif.Driver/Sdk/DriverEventNames.cs

```diff
@@ -9,10 +9,20 @@ public sealed class DriverEventNames
     {
         public const string ArtifactNotScanned = nameof(ArtifactNotScanned);
 
+        public const string EnumerateArtifacts = nameof(EnumerateArtifacts);
+        public const string EnumerateArtifactsStop = $"{EnumerateArtifacts}/Stop";
+        public const string EnumerateArtifactsStart = $"{EnumerateArtifacts}/Start";
+
         public const string ReadArtifact = nameof(ReadArtifact);
         public const string ReadArtifactStop = $"{ReadArtifact}/Stop";
         public const string ReadArtifactStart = $"{ReadArtifact}/Start";
 
+        public const string Scanned = nameof(Scanned);
+
+        public const string ScanArtifact = nameof(ScanArtifact);
+        public const string ScanArtifactStop = $"{ScanArtifact}/Stop";
```

Comment:
>Stop [](http://example.com/codeflow?start=64&length=4)

Any value in doing a public const string StartEvent = nameof(Start), and a similar one for Stop?

Reply:
Sure, I think so, simply because it's repeated everywhere. These terms aren't actively referenced separately but using a constant will protect against typos. Next change! Thanks for the note.